lib/csv_class_maker/csv_find.rb

```diff
@@ -70,9 +94,16 @@ def each
       end
     end
 
-
     private
 
+    def extract_headers(file_name, options)
+      csv_file = File.open(file_name,'r')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.